### PR TITLE
Differences between .13 <-> 1.0, add influx_inspect doc

### LINKED ISSUE #543: Add documentation for influx_inspect export tool
Update the documentation based on the readme for the `influx_inspect` tool, specifically the export functionality.

https://github.com/influxdata/influxdb/pull/7046

### PR DESCRIPTION
**Pull request by corylanou**
fixes #543

### CODE REVIEW

File: content/influxdb/v1.0/administration/013_vs_1.md [hunk 1]
```diff
@@ -12,11 +12,23 @@ See [InfluxDB's Changelog](https://github.com/influxdata/influxdb/blob/master/CH
...
+* [Breaking Changes](#breaking-changes)
 * [`SHOW FIELD KEYS` returns fields' data types](#show-field-keys-returns-fields-data-types)
 * [Modified name for the auto-generated retention policy](#modified-name-for-the-auto-generated-retention-policy)
 * [New InfluxQL syntax introduces casting and differentiating between identical field and tag keys](#new-influxql-syntax-introduces-casting-and-differentiating-between-identical-field-and-tag-keys)
 * [InfluxQL function updates](#influxql-function-updates)
 * [Logging change](#logging-change)
+* [Add support for JWT token authentication](#jwt-token-authentication)
+* [Add tsm file export to influx_inspect tool](#influx-inspect-export)
+
+### Breaking Changes
```
Comment (corylanou) [inline content/influxdb/v1.0/administration/013_vs_1.md]: I'm not sure if you want this section or not. Feel free to delete if you think it's too specific for the docs.

Comment (beckettsean) [inline content/influxdb/v1.0/administration/013_vs_1.md]: I like it! Thanks for the breaking changes notes. I made a few direct commits to them, let me know if I broke the meaning.

**Comment by beckettsean** [comment]
This is great, @corylanou, thanks!

I made a few edits to this but I think it's ready to merge. 

@rkuchan?

**Comment by rkuchan** [comment]
@sean, I made a couple changes. I removed the sentence about being able to configure the name of the default retention policy because that seems to have been removed from more recent versions.

**Pull request merged**